src/no/runsafe/framework/api/entity/animals/horses/IAbstractHorse.java

```diff
@@ -0,0 +1,19 @@
+package no.runsafe.framework.api.entity.animals.horses;
+
+import no.runsafe.framework.api.entity.IAnimal;
+import no.runsafe.framework.api.entity.ITameable;
+
+public interface IAbstractHorse extends IAnimal, ITameable
```

Comment:
Maybe call them IHorse, IRideableHorse and IChestedHorse? .. though I'm not super happy about the gramatics of IChestedHorse, I can live with that, I suppose